zcash_test_vectors/transaction_v6.py

```diff
@@ -81,6 +81,38 @@ def __bytes__(self):
         return ret
 
 
+class ActionGroupDescription(object):
+    def __init__(self, rand, anchor_orchard, proofs_orchard, is_coinbase):
+        self.vActionsOrchard = []
+        # There must always be a non-zero number of Action Descriptions in an Action Group.
+        for _ in range(rand.u8() % 4 + 1):
+            self.vActionsOrchard.append(OrchardZSAActionDescription(rand))
+        self.flagsOrchard = rand.u8()
+        # Three flag bits are defined, we set enableZSA to true.
+        self.flagsOrchard = (self.flagsOrchard & 7) | 4
+        if is_coinbase:
+            # set enableSpendsOrchard = 0
+            self.flagsOrchard &= 2
```

Comment:
do we have a test for a coinbase transaction?

Reply:
this will disable the third bit, isn't it?

Reply:
Yes, we need the third bit (`enableZSAs`) disabled for coinbase transactions.

Reply:
ok, do we have a test for a coinbase transaction?

Reply:
There are two test vectors (7th and 9th) that are set to be coinbase transactions.